.eslintrc

```diff
@@ -1,42 +1,46 @@
 {
-  "parser": "@babel/eslint-parser",
```

Comment:
i guess it is fine, if it does not lead to linting issues? I guess we need this part of eslint-config for jsx/js files?
And the part below is the config for tsx/ts files?

Reply:
yeah, using eslint on a mixed code-base is a bit hacky. I wasn't sure if setting the ecma version to 13 is a problem but since we use babel to transpile the code during build anyway it shouldn't matter